packages/serverless-helpers/helpers/testFunctionNames.ts

```diff
@@ -16,6 +16,13 @@ const getFullFunctionName = (config: AWS, functionName: string): string =>
  */
 export const testFunctionNames = (config: AWS): void => {
   const functionNames = getFunctionNames(config);
+  if (functionNames.length === 0) {
+    it('There are no functions declared', () => {
```

Comment:
We can keep the same syntax across all tests, and use
```suggestion
    it('`has no functions declared`', () => {
```